src/fw/services/normal/filesystem/pfs.c

```diff
@@ -1491,8 +1491,10 @@ status_t pfs_remove(const char *name) {
   status_t rv = get_avail_fd(name, &fd, false);
   if (rv >= FDAlreadyLoaded) { // the file is in the cache
     if (rv == FDBusy) {
-      PBL_CROAK("Cannot delete %s, it is currently in use",
-                PFS_FD(fd).file.name);
+      PBL_LOG(LOG_LEVEL_WARNING, "Cannot delete %s, it is currently in use",
+              PFS_FD(fd).file.name);
+      rv = E_BUSY;
+      goto cleanup;
```

Comment:
My $0.02: we absolutely *MUST NOT* remove the `CROAK` in an error case that indicates system inconsistency.  If something is specified to never happen, other parts of the system may depend on that not happening, and that might result in consequences more dire than this particular case if it does.